devchat.go

```diff
@@ -279,6 +279,16 @@ func newUser(s ssh.Session) *user {
 	return u
 }
 
+// Deletes any empty room. Checks all rooms so that it works even when the
+// user left.
+func cleanupRooms() {
+	for _, r := range rooms {
```

Comment:
We only need to check the room this user left, so a loop through all rooms is not necessary. I would have this function take in a room argument to check.